lib/Models/WebProcessingServiceCatalogItem.js

```diff
@@ -27,6 +28,7 @@ function WebProcessingServiceCatalogItem(terria) {
     CatalogItem.call(this, terria);
 
     this._geoJsonItem = undefined;
+    this._terria = terria;
```

Comment:
This isn't necessary, the `CatalogItem` base class exposes terria on the `terria` property itself already.